pkg/db/tx_test.go

```diff
@@ -3,10 +3,9 @@ package db_test
 import (
 	"context"
 	"errors"
-	"testing"
-
 	"github.com/treeverse/lakefs/pkg/db"
 	"github.com/treeverse/lakefs/pkg/db/params"
+	"testing"
```

Comment:
Use goimports it will take "testing" up with the rest of the packages and not here